landing-pages/site/content/en/blog/documenting-using-local-development-environments.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Documenting using local development environment"
+linkTitle: "Documenting using local development environment"
+author: "Elena Fedotova"
+github: "efedotova"
+linkedin: "elena-fedotova-039294110"
+description: "The story behind documenting local development environment of Apache Airflow"
+tags: ["Development"]
+date: 2019-11-22
+---
+
+## Documenting local development environment of Apache Airflow
+
+From Sept to November, 2019 I have been participating in a wonderful initiative, Google Season of Docs.
+
+I had a pleasure to contribute to the Apache Airflow open source project as a technical writer.
+My initial assignment was an extension to the github-based Contribution guide.
+
+From the very first days I have been pretty closely involved into inter-project communications
+via emails/slack and had regular 1:1s with my mentor, Jarek Potiuk.
+
+I got infected with Jarek’s enthusiasm to ease the on-boarding experience for
+Airflow contributors. I do share this strategy and did my best to improve the structure,
+language and DX. As a result, Jarek and I extended the current contributor’s docs and
+ended up with the Contributing guide navigating the users through the project
+infrastructure and providing a workflow example based on a real-life use case;
+the Testing guide with an overview of a complex testing infrastructure for Apache Airflow;
+and two guides dedicated to the Breeze dev environment and local virtual environment
+(my initial assignment).
+
+I’m deeply grateful to my mentor and Airflow developers for their feedback,
+patience and help while I was breaking through new challenges
+(I’ve never worked on an open source project before),
+and for their support of all my ideas! I think a key success factor for any contributor
+is a responsive, supportive and motivated team, and I was lucky to join such
+a team for 3 months.
+
+Documents I worked on:
+
+* https://github.com/apache/airflow/blob/master/BREEZE.rst
```

Comment:
Can you use Use meaningful link text?
For more information: 
https://developers.google.com/style/cross-references